components/molecule/drawer/src/index.scss

```
@@ -90,6 +90,78 @@ $bgc-molecule-drawer-overlay-finish: rgba(0, 0, 0, 0.6) !default;
   }
 }
 
+@keyframes open-drawer-left--fullscreen {
+  0% {
+    left: -100%;
+  }
+  100% {
+    left: 0px;
```

We don't need to define a unit to the value zero.
Same at the rest of the code.

```suggestion
    left: 0;
```